include/fizzy/fizzy.h

```diff
@@ -148,14 +173,23 @@ FizzyInstance* fizzy_instantiate(const FizzyModule* module,
 /// @param      imported_functions       Pointer to the imported function array. Can be NULL iff
 ///                                      imported_functions_size equals 0.
 /// @param      imported_functions_size  Size of the imported function array. Can be zero.
+/// @param      imported_globals         Pointer to the imported globals array. Can be NULL iff
+///                                      imported_globals_size equals 0.
+/// @param      imported_globals_size    Size of the imported global array. Can be zero.
 /// @returns    non-NULL pointer to instance in case of success, NULL otherwise.
 ///
 /// @note
 /// Functions in @a imported_functions are allowed to be in any order and allowed to include some
 /// functions not required by the module.
 /// Functions are matched to module's imports based on their module and name strings.
+/// @note
+/// Function expects @a imported_globals to be in the order of imports defined in the module.
```

Comment:
It's weird how resolve_instantiate currently doesn't resolve globals the way it resolves functions. C++ backend doesn't allow this currently, and I don't know how important it is.

Reply:
I agree there should be a function similar to functions to resolve globals in the C++ API. That could be used here.

Probably at that point I'd rename the to instantiates as `fizzy_instantiate` (which does resolution) and `fizzy_unsafe_instantiate` (which doesn't).